README.md

```diff
@@ -108,7 +115,11 @@ kubectl --kubeconfig=<(microk8s config) create secret docker-registry gcr-access
 
 ## Deploying to local cluster
 
-```
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami # Add the helm repo for RabbitMQ
+helm repo add elastic https://helm.elastic.co # Add the helm repo for Kibana
+helm repo add minio https://charts.min.io/ # Add the helm repo for MinIO
+
```

Comment:
nice to have here
But we need to find a different way to manage this (probably with docker and makefile)

Reply:
Sure, I just wanted it to work on my machine